### PR TITLE
videojs-standard Utils

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Makes `src/js/utils/` pass videojs-standard.

Previous discussion in #3460 

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/utils/log.js [hunk 1]
```diff
@@ -68,9 +71,9 @@ export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 1
  *
  * @function log
  */
-function log(...args) {
+log = function(...args) {
```
Comment (gkatsev) [inline src/js/utils/log.js]: we may want to keep this as `function log` for the whole IE8 shenanigans.

File: src/js/utils/merge-options.js [hunk 2]
```diff
@@ -41,7 +43,7 @@ const customizer = function(destination, source) {
  * provided objects
  * @function mergeOptions
  */
-export default function mergeOptions() {
+mergeOptions = function() {
```
Comment (gkatsev) [inline src/js/utils/merge-options.js]: keep this as it was for IE8...

**Comment by gkatsev** [comment]
Two issues for IE8 compat.
LGTM otherwise.

File: src/js/utils/log.js (hunk 1 above)
Comment (misteroneill) [inline src/js/utils/log.js]: Yeah. I did it this way because it conflicts with `no-use-before-define`.

File: src/js/utils/merge-options.js (hunk 2 above)
Comment (misteroneill) [inline src/js/utils/merge-options.js]: Yeah. I did it this way because it conflicts with no-use-before-define.

Comment (misteroneill) [inline src/js/utils/merge-options.js]: Why is this a problem in IE8?

Comment (gkatsev) [inline src/js/utils/merge-options.js]: Guess it doesn't need to change.
I was looking for changes from `function foo` to `foo = function` and got a bit overzealous.

**Pull request merged**